assets/src/dashboard/components/index.js

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
```

Comment:
some of these were just not in alphabetical order, which I arranged so that it looked like the file structure, just for my own sanity. the removed ones were not used anymore. I was able to remove the typography module completely which had most of the `DEPRECATED_THEME` usage.